core-tests/src/test/java/org/mule/runtime/core/internal/construct/DefaultFlowBuilderTestCase.java

```diff
@@ -71,7 +74,7 @@ public void buildsSimpleFlow() throws Exception {
     assertThat(flow.getMuleContext(), is(muleContext));
     assertThat(flow.getProcessors(), is(empty()));
     assertThat(flow.getSource(), is(nullValue()));
-    assertThat(flow.getExceptionListener(), sameInstance(muleContext.getDefaultErrorHandler()));
+    assertThat(flow.getExceptionListener(), not(sameInstance(muleContext.getDefaultErrorHandler(Optional.empty()))));
```

Comment:
No. Each flow needs it's own copy of the error handler. Take into account that error handler and the processors inside it support lifecycle. If each of them have the same copy then it means that if flow A and B are using the default error handling, doing stop over A will stop MPs inside B error handler